example/cursor/index.html

```diff
@@ -61,7 +61,7 @@ <h1 itemprop="name">Cursor Plugin Example</h1>
 <pre><code>var wavesurfer = WaveSurfer.create({
     container: document.querySelector('#waveform'),
     plugins: [
-        WaveSurfer.cursor()
+        WaveSurfer.cursor.create()
```

Comment:
ill have to re-read your motivation but i'd def prefer cursor() over create(). This wouldn't work either would it:

```
var cursorPlugin = WaveSurfer.cursor.create();
...
plugins: [cursorPlugin]
...
```